src/journal-remote/journal-remote.c

```diff
@@ -919,6 +920,9 @@ static int remoteserver_init(RemoteServer *s,
                 else
                         url = strdupa(arg_url);
 
+                if (arg_follow)
+                        url = strjoina(url, "?follow=1");
+
```

Comment:
Ah I missed that case. I think I can fix this by checking for the presence of "/entries?" and appending "&follow=1" instead of that but seeing that case I now think this feature may be unneeded. What I am actually developing for is an older version of systemd where "/entries" was unconditionally appended whenever --url was passed. Now however it appears that I can get the same functionality by doing `--url='http://some.host:19531/entries?follow=1'`.